crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -1277,9 +1277,8 @@ impl CheckpointServiceNotify for CheckpointService {
     }
 }
 
-#[cfg(test)]
+// test helper
```

Comment:
The only thing that gives me pause to accept is whether this config change is okay, especially given that the reason to expose the test helpers is to use them in another test.

Reply:
Just a huge ball of knots. And no matter how hard I tried, I couldn't get the transactional test system to work under `cfg(test)`, which I think is the result of some oddities with datatest

Reply:
The only thing I use to calm myself about things like this, is we have soooo many worse things exposed outside of `cfg(test)`